docs/alerting/api.md

```diff
@@ -95,6 +95,73 @@ If you use a custom webhook for your destination and need to embed JSON in the m
 }
 ```
 
+If you want to specify a timezone, you can do so by including a [cron expression](./cron) with a `ZoneId` in the `schedule` section of your request.
+
+The following example creates a monitor for 12:10 PM US Pacific Time for the 1st of every month.
+
+```json
+{
+  "type": "monitor",
+  "name": "test-monitor",
+  "enabled": true,
+  "schedule": {
+    "cron" : {
+        "expression": "10 12 1 * *",
+        "timezone": "US/Pacific"
+    }
+  },
+  "inputs": [{
+    "search": {
+      "indices": ["movies"],
+      "query": {
+        "size": 0,
+        "aggregations": {},
+        "query": {
+          "bool": {
+            "filter": {
+              "range": {
+                "@timestamp": {
+                  "gte": "||-1h",
+                  "lte": "",
+                  "format": "epoch_millis"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }],
+  "triggers": [{
+    "name": "test-trigger",
+    "severity": "1",
+    "condition": {
+      "script": {
+        "source": "ctx.results[0].hits.total.value > 0",
+        "lang": "painless"
+      }
+    },
+    "actions": [{
+      "name": "test-action",
+      "destination_id": "ld7912sBlQ5JUWWFThoW",
+      "message_template": {
+        "source": "This is my message body."
+      },
+      "throttle_enabled": true,
+      "throttle": {
+        "value": 27,
+        "unit": "MINUTES"
+      },
+      "subject_template": {
+        "source": "TheSubject"
+      }
+    }]
+  }]
+}
+```
+
+The alerting plugin uses the Java `Time Zone` class to convert `ZoneId` to valid timezones for processing. For more information about `ZoneId`, refer to the [official documentation about the Time Zone class](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html).
```

Comment:
So a bunch of nitpicks on this sentence:

* I feel like whatever link we have here should let you know about all the time zone options in the event that you don't care about the Java portion of the backend. Based on where the Java documentation links to, that seems to be: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
* For Java, we have a strong preference for 11 over 8, so let's link to: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/TimeZone.html and https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/ZoneId.html.
* Java classes are always TitleCaseNoSpace as far as I know, so TimeZone.
* The current link says "here's the docs for Time Zone," but then it links to ZoneId.

Make changes as necessary, but something more like:

```
See [Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) for a full list of tz database timezone names. The alerting plugin uses the Java [TimeZone](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/TimeZone.html) class to convert from a [ZoneId](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/ZoneId.html) to a valid timezone.
```